### PR TITLE
(maint) Update modulesync

### PR DESCRIPTION
**Pull request by Iristyle**

### CODE REVIEW

**Comment by ferventcoder** [comment]
Once appveyor is happy, I'm happy. Did any of the other repositories get changes as well?

**Comment by Iristyle** [comment]
I was only concerned with DSC atm

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,7 @@
 pkg/
 Gemfile.lock
 Gemfile.local
+vendor/
```
Comment (Iristyle) [inline .gitignore]: This was previously removed in https://github.com/puppetlabs/puppetlabs-dsc/blob/25240bc5b4eae6cd9cc195856d012219053d762b/.gitignore ... and it looks like we'd have to modify the template to enact any change here: https://github.com/puppetlabs/modulesync_configs/blob/windows/moduleroot/.gitignore

Comment (ferventcoder) [inline .gitignore]: We'll need to adjust that one back out. I'll take care of that.

**Pull request merged**